.sync.yml

```diff
@@ -1,11 +1 @@
-.travis.yml:
-  env:
-    global:
-      - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
-      - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
```

Comment:
Need to double check if this plugin have any tests. This section removes everything besides build.